packages/language-server/src/server.ts

```diff
@@ -112,17 +110,16 @@ connection.onInitialize(async (params) => {
 			},
 		})
 		service.project
-			.on('documentErrored', ({ errors, uri, version }) => {
-				if (!core.fileUtil.isFileUri(uri)) {
-					// Don't send errors for non-file URIs to the client.
-					return
+			.on('documentErrored', async ({ errors, uri, version }) => {
+				try {
+					await connection.sendDiagnostics({
+						diagnostics: toLS.diagnostics(errors),
+						uri: uri,
+						version: version,
```

Comment:
i know you just moved this object and it's not actually new, but it made me think of [`object-shorthand`](https://eslint.org/docs/latest/rules/object-shorthand)

definitely more of a style-focused rule though. thoughts?

Reply:
yeah good call. i guess it's not really necessary to enable the lint rule since it's pretty minor but i do prefer the shorthand style as well.